src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -98,4 +105,52 @@ protected function compileEndIsset()
     {
         return '<?php endif; ?>';
     }
+
+    /**
+     * Compile the switch statements into valid PHP.
+     *
+     * @param  string  $variable
+     * @return string
+     */
+    protected function compileSwitch($variable)
+    {
+        return "<?php switch ({$variable}): ";
+    }
+
+    /**
+     * Compile the case statements into valid PHP.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    protected function compileCase($value)
+    {
+        if ($this->firstCaseInSwitch) {
+            $this->firstCaseInSwitch = false;
```

Comment:
@danharper Implemented the `firstCaseInSwitch` because of the following as worded in the PHP docs:

> Warning
Any output (including whitespace) between a switch statement and the first case will result in a syntax error

Trying to fix the issue you just found out about multiple switch statements in a file, any help or suggestions?